### PR TITLE
fix(compile): fixing gradle property for Dockerfile.compile

### PR DESCRIPTION
**Pull request by nimakaviani**
Fixing the typo in the compile script

@plumpy

### CODE REVIEW

**Review by plumpy — APPROVED** [review]
🥳

File: Dockerfile.compile [hunk 1]
```diff
@@ -19,4 +19,4 @@ RUN usermod -d /var/lib/mysql/ mysql && \
     mysql -u root --password=sa -e 'CREATE DATABASE `keel` DEFAULT CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;'
 
 CMD /etc/init.d/mysql restart && \
-    ./gradlew Plugin=true -PbuildingInDocker=true keel-web:installDist -x test
+    ./gradlew --no-daemon -PbuildingInDocker=true keel-web:installDist -x test
```
Comment (luispollo) [inline Dockerfile.compile:22]: I wonder what the original flag was intended for... Any ideas?

**Comment by plumpy** [comment]
It was just an editing mistake. It originally looked like this:
`--no-daemon -PenableCrossCompilePlugin=true`

I hit `33x` in vim to delete 33 characters (the length of the second argument) but my cursor was in the wrong place, so it deleted the wrong 33 characters 😳  And I'd done that same change in like 50 different places, so I was just moving too quickly and screwed it up...

**Comment by luispollo** [comment]
Haha! It happens! 😄 

Thanks for fixing it!

**Pull request merged**